src/test/groovy/org/prebid/server/functional/model/response/auction/InvocationResult.groovy

```diff
@@ -0,0 +1,18 @@
+package org.prebid.server.functional.model.response.auction
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import groovy.transform.ToString
+import org.prebid.server.functional.model.config.HookId
+import org.prebid.server.functional.model.request.auction.FetchStatus
+
+@ToString(includeNames = true, ignoreNulls = true)
+@JsonNaming(PropertyNamingStrategies.LowerCaseStrategy)
+class InvocationResult {
+
+    Long executionTimeMillis
+    FetchStatus status
```

Comment:
I'm not sure that it's the same type. FetchStatus was made for Floors AFAIK. Module invocation status is a different thing. At least as far as I remember.

Reply:
good catch, changed to a new object `InvocationStatus`